README.md

```diff
@@ -136,7 +136,7 @@ $ kubectl delete -f deploy/crds/app.example.com_appservices_crd.yaml
 
 To learn more about the SDK CLI, see the [SDK CLI Reference][sdk_cli_ref], or run `operator-sdk [command] -h`.
 
-## User Guides
+## User Guides for Operator SDK
```

Comment:
Revert. I think we're mostly worried about document titles, not section headers within the document.

```suggestion
## User Guides
```

Reply:
done 👍